tests/qnodes/test_qnode_cv.py

```diff
@@ -613,3 +613,30 @@ def circuit(a):
 
         with pytest.raises(ValueError, match=r"cannot be used with the argument\(s\) \{'a'\}"):
             circuit.jacobian([1.0], method="A")
+
+    def test_error_unsupported_grad_recipe(self, monkeypatch):
+        """Test exception raised if attempting to use the second order rule for
+        computing the gradient analytically of an expectation value that
+        contains an operation with an unsupported gradient recipe"""
```

Comment:
might be good to say why it isn't supported!
```suggestion
        contains an operation with an more than two terms in the gradient recipe"""
```